### PR TITLE
If bridge V3 don't run for binance and rinkeby and other fixes

### PR DESCRIPTION
**Pull request by pmprete**
Fix federator retrie condition
Clean log context after transaction
Combine logs to compact information on screen
Add chainId to IBridgeV3 constructor to filter Binance and Rinkeby from running
Add harcoded version after checking the version with the factory to reduce network calls
Fix ntfIntegrationTest db path

### CODE REVIEW

File: federator/src/contracts/IBridgeV3.ts [hunk 1]
```diff
@@ -21,6 +24,10 @@ export class IBridgeV3 implements IBridge {
   }
 
   async getPastEvents(eventName: string, destinationChainId: number, options: any): Promise<EventData[]> {
+    if ([4, 56, 97].includes(this.chainId)) {
```
Comment (aguschirico) [inline federator/src/contracts/IBridgeV3.ts:27]: Perhaps is a good idea to get the array of chain ids from a constant/s?

File: federator/src/lib/FederatorERC.ts [hunk 2]
```diff
@@ -88,7 +90,17 @@ export default class FederatorERC extends Federator {
...
     }
     fromBlock = fromBlock + 1;
-    this.logger.debug('Running from Block', fromBlock);
+
+    this.logger.info(
+      `Running ${fromBlock} to Block ${toBlock} Large Amount Confirmations: ${confirmations.largeAmountConfirmations}`,
+    );
+    const destinationBridge = await bridgeFactory.createInstance(sideChainConfig);
+    const destinationBridgeVersion = destinationBridge.getVersion();
+    if (destinationBridgeVersion == VERSIONS.V3 && [4, 56, 97].includes(sideChainConfig.chainId)) {
```
Comment (aguschirico) [inline federator/src/lib/FederatorERC.ts:99]: same here

**Review by aguschirico — APPROVED** [review]

File: federator/src/contracts/IBridgeV3.ts (hunk 1 above)
Comment (pmprete) [reply federator/src/contracts/IBridgeV3.ts:27]: Will do that 👍

File: federator/src/lib/FederatorERC.ts (hunk 2 above)
Comment (pmprete) [reply federator/src/lib/FederatorERC.ts]: done

**Pull request merged**